tests/test_data_preparation.py

```diff
@@ -54,24 +82,29 @@ def test_multiple_files():
 
 
 def test_exclude_keys():
-    output_file = "tests/data/processed_compact_output.jsonl"
-    subprocess.run(
-        "python -m nemo_skills.training.prepare_sft_data "
-        f"    ++input_files='tests/data/output-rs*.test' "
-        f"    ++output_path={output_file} "
-        f"    ++prompt_config=generic/math "
-        f"    ++prompt_template=llama3-instruct "
-        f"    ++exclude_optional_keys=true "
-        f"    ++filters.remove_len_outlier_problems=false "
-        f"    ++filters.drop_multi_boxed=true "
-        f"    ++filters.trim_solutions=true "
-        f"    ++filters.drop_incorrect_arithmetic=false "
-        f"    ++filters.split_arithmetic=false "
-        f"    ++num_output_samples=32 "
-        f"    ++downsampling_method=fair "
-        f"    ++do_shuffle=false ",
-        check=True,
-        shell=True,
+    output_file = f"/mnt/datadrive/nemo-skills-test-data/tests/data/processed_compact_output_{uuid.uuid4()}.jsonl"
```

Comment:
Can you please save inside `/tmp/nemo-skills-tests` instead? The datadrive location is a mounted blob storage, so we don't want to keep putting new files there. Also probably better to remove the file if it exists before the test rather than using uuid as it would be hard to find the relevant file out of many there

Reply:
done: https://github.com/Kipok/NeMo-Skills/pull/240/files/f17dfac56ee190a48bac3036dd820ab1f4023746..2794f41708fb390ea3c8a613bdc4495f98155a90